core/src/js/components/mercenaries/overlay/teams/mercenaries-team-ability.component.ts

```diff
@@ -77,6 +79,7 @@ export class MercenariesTeamAbilityComponent {
 		const abilityCard = this.allCards.getCard(value.cardId);
 		this.type = value.type;
 		this.cardId = value.cardId;
+		this.relatedCardIds = abilityCard.relatedCardDbfIds?.map(String) ?? [];
```

Comment:
I think this won't work, as the relatedCardDbfIds return dbfIds (numbers), and the relatedCards tooltip expects cardIds. So you probably will need to map the dbfId to a standard cardId

Reply:
`abilityCard.relatedCardDbfIds?.map(dbfId => this.allCards.getCard(dbfId).id) ?? []`